functions/offset.py

```diff
@@ -0,0 +1,41 @@
+def offset(proposed_mode,chosen_mode):
+   """
+   Function takes proposed mode as one argument which is a dictionary of modes of transport based on distance,
+   chosen mode as another argument which is a dictionary of one selected mode by user for that journey
+   :param proposed_mode:
+   :param chosen_mode:
+
+   """
+   prop_trans=max(proposed_mode, key=proposed_mode.get)
+   prop_emission_max= proposed_mode.get(prop_trans)
+   #print(prop_emission_max)
+   for key in chosen_mode:
+     chose_emission= chosen_mode.get(key)
+   #print(chose_emission)
+   def emission_return(prop_emission_max,chose_emission):

```

Comment:
I don't think you should use a nested function. Also, I don't think you need the if statement we should only do the print statements in the user interface (in our main).

The function offset should only return the value offset.